h2o-automl/src/main/java/ai/h2o/automl/AutoML.java

```diff
@@ -638,8 +638,13 @@ private void setCommonModelBuilderParams(Model.Parameters params) {
       params._weights_column = buildSpec.input_spec.weights_column;
 
       if (buildSpec.input_spec.fold_column == null) {
-        params._nfolds = 5;
-        params._fold_assignment = Model.Parameters.FoldAssignmentScheme.Modulo;
+        //params._nfolds = 5;
+        params._nfolds = buildSpec.build_control.nfolds;
+        if (buildSpec.build_control.nfolds > 1) {
+          // TO DO: below allow the user to specify this (vs Modulo)
```

Comment:
These `TO-DO`'s should be JIRAs if they are not already.

Reply:
This should be fixed when we expose `fold_assignment` to the clients.  In this JIRA: https://0xdata.atlassian.net/browse/PUBDEV-5039

Reply:
Great. Ty